staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1474,6 +1474,12 @@ const (
 	RunStrategyOnce VirtualMachineRunStrategy = "Once"
 )
 
+type UpdateVolumesStrategy string
+
+const (
+	UpdateVolumesStrategyReplacement UpdateVolumesStrategy = "Replacement"
```

Comment:
I think it is technically frowned upon to add new values to "enum" types so should we add "Hotplug" now?  Shouldn't matter if there is no release before hotplug is added

Reply:
mmh I could add "Migrate"

Reply:
Do you have concerns about "Hotplug"?

Reply:
I feel uncomfortable to add options not covered by the PR itself. For "Migrate",it will follow this PR and there is already an open draft PR with the option.

Reply:
I've actually started implementing hotplug but to be consistent I guess let's just keep this as-is.  Migrate and Hotplug should get in before we do a release so shouldn't be an issue

Reply:
This won't be the only case too. See the [StartStrategy](https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/api/core/v1/types.go#L73-L77)

Reply:
> But then I would rather prefer to just keep one value even if it sound a little weird and add the rest with the actual implementation

I think we are on the same page but just to confirm.  This PR will only have "Replacement" right?

Reply:
> > But then I would rather prefer to just keep one value even if it sound a little weird and add the rest with the actual implementation
> 
> I think we are on the same page but just to confirm. This PR will only have "Replacement" right?

Yes